e2e/tests/negative/replica_rebuilding.robot

```diff
@@ -66,6 +66,20 @@ Reboot Replica Node While Replica Rebuilding
         And Check volume 0 data is intact
     END
 
+Reboot Replica Node While Offline Replica Rebuilding
+    [Documentation]    Reboot a replica node during offline replica rebuilding is in progress.
+    ...
+    ...                Issue: https://github.com/longhorn/longhorn/issues/8443
+    ...    1. Create a volume and write some data to the volume.
+    ...    2. Detach the volume.
+    ...    3. Delete a replica of the volume.
+    ...    4. Enable offline rebuilding of the volume by the API `volume.offlineReplicaRebuilding`.
+    ...    5. Check if offline rebuilding starts.
+    ...    6. Reboot a replica node.
+    ...    7. Check if the volume offline rebuilding is in progress.
+    ...    8. The volume is detached after healthy replicas count of the volume equals to `Volume.Spec.NumberOfReplicas`.
+    Given Create volume 0 with    size=5Gi    numberOfReplicas=3    dataEngine=${DATA_ENGINE}
```

Comment:
It looks like the test case only includes a single Given step. Is this finished? What is this test intended to verify?

Reply:
> It looks like the test case only includes a single Given step. Is this finished? What is this test intended to verify?

I think @mantissahz  plans to add the `robot:skip` tag and then provide the  skeletons for negative test scenarios, which is not yet completed.

Reply:
Added the test steps for the negative test case.